docs/v2/guides/twig-filters.md

```diff
@@ -0,0 +1,234 @@
+---
+title: "Twig filters"
+order: "220"
+---
+
+## General Filters
+
+Twig offers a variety of [filters](http://twig.symfony.com/doc/filters/index.html) to transform text and other information into the desired output. In addition, Timber has added some valuable custom filters for your WordPress theme:
+
+## array
```

Comment:
Not sure how this renders, but since these are specific code elements what about:

## `array`

## `excerpt`

... etc.